pub/health.go

```diff
@@ -4,6 +4,7 @@ import (
 	"encoding/json"
 	"net/http"
 
+	"github.com/gin-gonic/gin"
```

Comment:
Nice library you found as alternative to gorilla (which is not maintaned for some time). Has nice design, seems to be actively supported, and is on MIT.